.husky/pre-commit

```diff
@@ -1,4 +1,19 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+DIR=$(pwd)
+echo $(php --version)
+
+# Switch to MAKEFILE.
+
+if [[ "$DIR" == *"/public/themes/contrib/"* ]]; then
```

Comment:
Maybe we should check if the tools are available from Drupal before continuing to use them?